stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -140,14 +190,23 @@ internal class PaymentOptionsViewModel(
                 )
             } ?: PrefsRepository.Noop()
 
+            val paymentMethodsRepository = PaymentMethodsRepository.Api(
+                stripeRepository = stripeRepository,
+                publishableKey = publishableKey,
+                stripeAccountId = stripeAccountId,
+                workContext = Dispatchers.IO
```

Comment:
We should probably make this a `Factory` constructor argument

Reply:
What are your thoughts on factory vs. constructors when to use one vs another?

Reply:
Is this because you don't want to expose the workContext?

Reply:
This will be addressed in a later PR